blueprints/serverless/cloud-run-explore/variables.tf

```diff
@@ -0,0 +1,55 @@
+variable "project_id" {
```

Comment:
you'll have to sort variables alphabetically

Reply:
Really? :D

Reply:
Also in outputs.tf?

Reply:
yes :)

Reply:
Answering to myself, re-reading Contributing...